mender-update/standalone.hpp

```diff
@@ -107,7 +107,8 @@ error::Error SaveStateData(database::KeyValueDatabase &db, const StateData &data
 
 error::Error RemoveStateData(database::KeyValueDatabase &db);
 
-ResultAndError Install(context::MenderContext &main_context, const string &src);
+ResultAndError Install(
+	context::MenderContext &main_context, const string &src, bool verify_signature = true);
```

Comment:
Why not simple reuse the `enum class Signature` here?

Reply:
I pondered it, but it would mean adding an `artifact` dependency to the `standalone` file. 

I'm not against it at all, actually pro, but didn't know if you'd consider it poor style.

Reply:
`standalone` already "sits higher" in the hierarchy of dependencies, so I think it's ok. In the `.cpp` file I believe we are already using it.